docs/usage/tables/parquet.md

```diff
@@ -0,0 +1,120 @@
+# Parquet
+
+This page explains how you can work with Parquet tables in your Unity Catalog. It will also explain the advantages and drawbacks of working with Parquet.
+
+## Set up Unity Catalog
+
+To follow along, make sure you have a local instance of Unity Catalog running by launching the following command from a terminal window:
+
+```sh
+bin/start-uc-server
+```
+
+This local UC server will come with some sample data pre-loaded.
+
+You can list all of the tables in your Unity Catalog using:
+
+```sh
+bin/uc table list --catalog unity --schema default
+```
+
+Your output should look something like this:
+
+```
+┌────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┐
```

Comment:
Could you modify the output so it shows the name of the table property and perhaps reduces the column widths of all other values?